scilpy/tests/arrays.py

```diff
@@ -6,420 +6,420 @@
 # in the FiberCup. Order 8, descoteaux07 basis.
 fodf_3x3_order8_descoteaux07 = np.asarray([[[
```

Comment:
Is this the same data as in the verifications in scil_sh_to_aodf? Are you doing twice the same test, once as unit test and once in the scripts tests? If so, the unit test could be removed if you wish, and you just add a comment to see the script test. If different, could you still add a comment in the unit test to reference to the script test and explain the difference?

Reply:
this data is only used in the unit tests while the script test uses a nifti image and tests the script input/output. I'll add a note

Reply:
I meant: did you open the nifti to see its values and copy them here? Or they are entirely other values?

Reply:
Entirely other values, they are from the fibercup while the nifti is a whole brain dataset.